src/BibleWell/Models/TipTapModel.cs

```diff
@@ -0,0 +1,92 @@
+using System.Text.Json.Serialization;
+
+namespace BibleWell.Models;
```

Comment:
`BibleWell` by definition _is_ business models and interfaces so having a `Models` sub-namespace is unnecessary.  I'd move this to `BibleWell/Aquifer/TiptapModel.cs` (note the file name casing change as well).